js/bit4you.js

```diff
@@ -0,0 +1,624 @@
+                        'market/orderbook',
+                    ],
+                    'get': [
+                        'market/assets',
+                        'market/list',
+                        'market/summaries',
+                        'udf/history', // ?iso={iso}&resolution={resolution}&from={from}&to={to}
+                    ],
+                },
+                'private': {
+                    'post': [
+                        'wallet/balances',
+                        'wallet/blockchain-history',
+                        'wallet/send',
+                        'portfolio/history',
+                        'order/pending',
+                        'order/info',
+                        'order/list',
+                        'order/create',
+                        'order/cancel',
+                    ],
+                    'get': [],
+                },
+            },
+            'timeframes': {
+                '1m': '1m',
+                '5m': '5m',
+                '15m': '15m',
+                '30m': '30m',
+                '1h': '1h',
+                '4h': '4h',
+                '1d': '1d',
+            },
+            'urls': {
+                'logo': 'https://www.bit4you.io/img/logo/logo.svg',
+                'www': 'https://www.bit4you.io/',
+                'doc': 'https://docs.bit4you.io/',
+                'fees': 'https://www.bit4you.io/services#fees',
+                'api': {
+                    'coinmarketcap': 'https://www.bit4you.io/api/cmc/v1/', // ex:"/api/cmc/v1/summary"
+                    'graphql': 'https://www.bit4you.io/api/markets/graphql',
+                    'url': 'https://www.bit4you.io/api',
+                },
+            },
+        });
+    }
+
+    async fetchOHLCV (symbol, timeframe = '1h', since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        // console.log (market)
+        const request = {
+            'symbol': market['symbol'].replace ('/', '-'),
+            'resolution': this.timeframes[timeframe],
+            'from': since,
+            'to': params && params.to || null,
+        };
+        try {
+            const response = await this.publicGetUdfHistory (this.extend (request, params));
+            return response;
+        } catch (error) {
+            return error;
+        }
+    }
+
+    async fetchMarkets (params = {}) {
+        try {
+            const markets = await this.publicGetMarketList ();
+            const marketIds = Object.keys (markets);
+            const result = [];
+            for (let i = 0; i < marketIds.length; i++) {
+                const marketId = marketIds[i];
+                const market = this.safeValue (markets, marketId);
+                const id = this.safeString (market, 'iso').replace ('-', '').toLowerCase ();
+                const symbol = this.safeString (market, 'iso').replace ('-', '/').toUpperCase ();
```

Comment:
On the other hand, symbol (our unified representation) has to be constructed with the unified base and quote 

Check other exchanges for reference, for instance: https://github.com/ccxt/ccxt/blob/master/js/hollaex.js#L236